### PR TITLE
Docs fixes and cleanup and addition

### PR DESCRIPTION
**Pull request by x5engine**
I have edited the description of the readme and deleted any not clear things and added more clarification with more links to join Gitcoin Community on Discord and Slack... and have made it simple for most people to understand and get started with the platform :)

### CODE REVIEW

**Comment by owocki** [comment]
what problem does this PR solve?  it seems to just be formatting changes.

**Comment by x5engine** [comment]
@owocki there were many links dead in the table content and also the intro should have been in the top and many changes to make it easier to read

**Comment by x5engine** [comment]
I check all the links here and on the other docs pages too

**Comment by x5engine** [comment]
@owocki can you please review this? thanks

**Comment by owocki** [comment]
this LGTM.  @danlipert ?

File: docs/README.md [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+
 ```stable``` branch - live on gitcoin.co
 
 ## Documentation
 
 [https://docs.gitcoin.co](https://docs.gitcoin.co)
 
-## How to interact with this repo
+## Join us!
+Come and talk with us at our very own [Discord server](https://discordapp.com/invite/QRA2rXp)!
```
Comment (thelostone-mc) [inline docs/README.md]: Mind leaving an empty line here ?

File: docs/README.md [hunk 2]
```diff
@@ -40,28 +47,17 @@ This is the website that is live at [gitcoin.co](https://gitcoin.co)
...
+  * [Bounty](#bounty)
+  * [Tip or Kudos](#tip-or-kudos)
+- [Licenses](#Licenses)
 
 ## What is Gitcoin?
 
-Functionally, the app has several key features:
+Gitcoin is an open source bounties platform on the Ethereum blockchain. We facilitate a space that allows open source developers to get paid for their work contributing to open source projects and in return, the open source projects get exposure to a vast community of hard working developers they might not have had otherwise.
+
+## Functionally
```
Comment (thelostone-mc) [inline docs/README.md]: Could we make it `###` as it's a subcategory of `What is Gitcoin` ?

File: docs/README.md [hunk 3]
```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
 
 [More about how/why to interact with web3 here](https://gitcoin.co/web3).
 
-Technically, the system is architected:
+## Technically
```
Comment (thelostone-mc) [inline docs/README.md]: Could we make it `###` as it's a subcategory of `What is Gitcoin` ?a

File: docs/README.md [hunk 4]
```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
...
 * __Web2__ This part of the app is built with Python, Django, Postgres, and a handful of other tools that are common in the web2 ecosystem.
 * __Web 3 Bridge__ This is the bridge between web3 and the rest of the application. Mostly built in javascript(web3js) and python(web3py).
 * __Brochureware__ Just a nice little landing page telling folks what the Gitcoin project is.
 
+
+## How to interact with this repo
+
+### On Github
+
+[Star](https://github.com/gitcoinco/web/stargazers) and [watch](https://github.com/gitcoinco/web/watchers) this github repository to stay up to date, we're pushing new code several times per week!
```
Comment (thelostone-mc) [inline docs/README.md]: Oh nicee 🙌

File: docs/README.md [hunk 5]
```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
...
+
+Check out the [CHANGELOG](https://docs.gitcoin.co/mk_changelog/) for details about recent changes to this repository.
```
Comment (thelostone-mc) [inline docs/README.md]: Could we replace this with releases https://github.com/gitcoinco/web/releases ?
Our changelog is captured there

File: docs/README.md [hunk 6]
```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
...
 
 If you
 
-* have some work on your Github Issues board that you don't have time (or skills) to do.
-* are dependant upon an upstream repo for something, and you'd like to incentivize them to work onit.
+* Have some work on your Github Issues board that you don't have time (or skills) to do.
+* Are dependant upon an upstream repo for something, and you'd like to incentivize them to work onit.
 
-try posting a funded issue at [https://gitcoin.co/new](https://gitcoin.co/new).
+Try posting a funded issue at [https://gitcoin.co/new](https://gitcoin.co/new).
```
Comment (thelostone-mc) [inline docs/README.md]: *Try funding your issue

**Comment by thelostone-mc** [comment]
@x5engine Oh do you mind updating the description
This ensures that reviewers can exactly tell what the PR before even checking the changes.

It's something I've learned by contributing to other open-source repos and hence the reason why I encourage folks to do the same

**Comment by x5engine** [comment]
hello @thelostone-mc thank you for reviewing my PR, I really appreciate it :)

I have updated the description and wish I could get paid for this bounty. thank you so much :)

**Comment by x5engine** [comment]
Thanks @owocki very generous.

**Comment by danlipert** [comment]
@x5engine looks good! can you resolve the conflicts and then I think we are good to merge

**Comment by thelostone-mc** [comment]
@x5engine would you wrapping this up anytime soon ?
If not, could we close this PR and open a draft PR instead

**Comment by x5engine** [comment]
hey @owocki @danlipert sure I will fix the conflicts and clean it up :)

File: docs/README.md (hunk 1 above)
Comment (x5engine) [reply docs/README.md]: added it :)

File: docs/README.md (hunk 2 above)
Comment (x5engine) [reply docs/README.md]: done also Technically is sub of what's Gitcoin

File: docs/README.md (hunk 3 above)
Comment (x5engine) [reply docs/README.md]: done

File: docs/README.md (hunk 4 above)
Comment (x5engine) [reply docs/README.md]: thanks :D

File: docs/README.md (hunk 5 above)
Comment (x5engine) [reply docs/README.md]: changed

File: docs/README.md (hunk 6 above)
Comment (x5engine) [reply docs/README.md]: done

**Comment by x5engine** [comment]
changes done :)

**Comment by x5engine** [comment]
@octavioamu @thelostone-mc can you review this please :) thank you

**Comment by thelostone-mc** [comment]
@x5engine I'm not sure if the code was pushed at your end 😅 
the conflicts are still present

**Comment by x5engine** [comment]
@thelostone-mc  https://github.com/gitcoinco/web/pull/4973/commits/c41c2685b04ebab385fdacc5bdd8d55550122ff2

isnt this good? :D

**Comment by thelostone-mc** [comment]
@x5engine your changes are good but i'm talking about the conflicts! coukd you resolve those ?

Screenshot 2019-10-23 at 5 01 02 AM

**Comment by thelostone-mc** [comment]
Closing this out as we couldn't get the conflicts resolved! 
Feel free to open another PR when you get a chance to resolve the conflicts
 @x5engine

**Pull request closed without merging**